example-zowe.yaml

```diff
@@ -351,7 +357,8 @@ zowe:
   #   ZWED_TN3270_PORT: 23
 
   # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-  # You can define any Zowe message portions to be checked for and the whole message added to the system log 
+  # You can define any Zowe message portions to be checked for and the message added to the 
+  # system log upon its logging, truncated to 512 characters. 
```

Comment:
no, 126. 512 was an in progress PR that never got committed that way, it's been 126 the whole time before & after the merge.